docker/base/build.sh

```diff
@@ -1,16 +1,16 @@
 cp /etc/apt/trusted.gpg.d/jetson-ota-public.asc ../.. # copy to jetbot root
 
-if [[ "$L4T_VERSION" == "32.4.3" ]]
-then
-    BASE_IMAGE=nvcr.io/nvidia/l4t-pytorch:r32.4.3-pth1.6-py3
-elif [[ "$L4T_VERSION" == "32.4.4" ]]
-then
-    BASE_IMAGE=nvcr.io/nvidia/l4t-pytorch:r32.4.4-pth1.6-py3
-fi
+BASE_IMAGE=$1
+
+echo "sudo docker build \
+    --build-arg BASE_IMAGE=$BASE_IMAGE \
+    -t jetbot/jetbot:base-$JETBOT_VERSION-$L4T_VERSION \
+    -f Dockerfile \
+    ../.."  # jetbot repo root as context
 
 sudo docker build \
     --build-arg BASE_IMAGE=$BASE_IMAGE \
-    -t $JETBOT_DOCKER_REMOTE/jetbot:base-$JETBOT_VERSION-$L4T_VERSION \
+    -t jetbot/jetbot:base-$JETBOT_VERSION-$L4T_VERSION \
```

Comment:
Is there a reason for removing JETBOT_DOCKER_REMOTE?

Reply:
Did not find this is actually being used, so this is just for simplifying things.

Reply:
It was used previously for hosting/sharing test containers with different remotes (ie: private ones).  I'm not sure if we'll need to do this anymore, but may be easiest / safest to leave it as is until we're sure we don't want to revert back.